iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/extensions/GeneratorShortCircuitXmlSerializer.java

```diff
@@ -40,7 +40,7 @@ public void write(GeneratorShortCircuit generatorShortCircuit, XmlWriterContext
 
     @Override
     public GeneratorShortCircuit read(Generator generator, XmlReaderContext context) throws XMLStreamException {
-        double directSubtransX = XmlUtil.readDoubleAttribute(context.getReader(), "directSubtransX");
+        double directSubtransX = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "directSubtransX");
```

Comment:
I actually just realized that there was a mistake in the Generator serializer. I do the fix here, is that ok?